.github/workflows/schedule.py

```diff
@@ -75,7 +89,105 @@ def _load_issue_template(path: str) -> tuple[dict[str, str], str]:
         return front_matter, f.read()
 
 
+def main(templates: list[tuple[str, datetime]], dry_run: bool) -> None:
+    for template, create_date in templates:
+        cmd_lookup, cmd_create = gh_commands(template, create_date)
+        if dry_run:
+            log.info('Would be running a mock command that creates an issue')
+            print(' '.join(cmd_create))
```

Comment:
log and print could go to different streams and mixing the two for a single "log event" is not a good idea.

Reply:
"Would be running a mock command" makes no sense.